openlibrary/templates/search/work_search_facets.html

```diff
@@ -72,12 +72,12 @@ <h4 class="facetHead">$(label)$:(magic_wand_markup)</h4>
             <span class="small"><a href="$facet_url" title="$link_title" data-ol-link-track="$add_track(header)">$display</a></span>&nbsp;<span class="smaller gray">$commify(count)</span>
     </div>
 
-<div id="searchFacets" data-config="$dumps({'start_facet_count': start_facet_count, 'facet_inc': facet_inc})">
+<div id="searchFacets" data-config="$dumps({'start_facet_count': start_facet_count, 'facet_inc': facet_inc})" data-param="$dumps(param)" data-async-load="$dumps(async_load)">
```

Comment:
It's awkward that `facet_counts=None` automatically turns async_load on ; sometimes it might be None because of an error or something. It would be clearer if `async_load` was an argument to this template.